assets/src/util/calendar-utils.js

```diff
@@ -1,4 +1,11 @@
 import dayjs from "dayjs";
+
+const internalOpeningHours = [];
+
+// Linting exceptions
+/* eslint no-underscore-dangle: 0 */
+/* eslint no-param-reassign: 0 */
```

Comment:
These are actually rules we would like to follow. So instead of ignoring them, fix the issues.

Reply:
moved no-param-reassign to concrete case, and added explanations for why both are used